src/components/Onboarding/index.js

```diff
@@ -110,7 +110,7 @@ const CloseContainer = styled(Box).attrs({
   p: 4,
   color: 'fog',
 })`
-  cursor: pointer;
+  cursor: initial;
```

Comment:
remove the line, so it inherit from parent

Reply:
done